### PR TITLE
Jquery ui button

### PR DESCRIPTION
**Pull request by rahulsayini**
UI Changes for Jwala operations page button 
  1. Updated start group/stop group button from un-bold
  2. Update css files to align expand collapse button for webserver and Jvm

### CODE REVIEW

**Review by jmahmoodcc — APPROVED** [review]

File: jwala-webapp/src/main/webapp/WEB-INF/views/jwala/index.jsp [hunk 1]
```diff
@@ -26,8 +26,6 @@
 <link rel="stylesheet" type="text/css" href="public-resources/css/adjustable-table-col.css">
 
 <link rel="stylesheet" type="text/css" href="public-resources/css/themes/${cookie['theme'] == null ? 'redmond' : cookie['theme'].value}/jquery-ui.min.css">
-<link rel="stylesheet" type="text/css" href="public-resources/css/themes/${cookie['theme'] == null ? 'redmond' : cookie['theme'].value}/jquery-ui.structure.min.css">
-<link rel="stylesheet" type="text/css" href="public-resources/css/themes/${cookie['theme'] == null ? 'redmond' : cookie['theme'].value}/jquery-ui.theme.min.css">
```
Comment (jeddcuison) [inline jwala-webapp/src/main/webapp/WEB-INF/views/jwala/index.jsp:30]: Why were these lines deleted ? I just want to know the reason.

Comment (jeddcuison) [reply jwala-webapp/src/main/webapp/WEB-INF/views/jwala/index.jsp:30]: Got you, getting a 404

File: jwala-webapp/src/main/webapp/resources/css/jquery-ui/accordion.css [hunk 2]
```diff
@@ -37,4 +37,16 @@
...
+}
+
+div.ui-accordion-header span.ui-accordion-header-icon.ui-icon.ui-icon-triangle-1-s {
+    float: left;
+    margin-left: -16px;
+    margin-top: 2px;
+}
+
+div.ui-accordion-header span.ui-accordion-header-icon.ui-icon.ui-icon-triangle-1-e {
+	float: left;
```
Comment (jeddcuison) [inline jwala-webapp/src/main/webapp/resources/css/jquery-ui/accordion.css]: Please align with the rest of the styles

**Review by jeddcuison — COMMENTED** [review]
Please align the styles in one of the css, please see my comments then I'll approve the pull request and merge it

**Review by jeddcuison — APPROVED** [review]
